src/bin/tikv-ctl.rs

```diff
@@ -1100,17 +1138,26 @@ fn main() {
         _ => unreachable!(),
     };
 
+    let mgr = new_security_mgr(&matches);
+
     if let Some(matches) = matches.subcommand_matches("dump-snap-meta") {
         let path = matches.value_of("file").unwrap();
         return dump_snap_meta_file(path);
+    } else if let Some(matches) = matches.subcommand_matches("compact-whole-cluster") {
+        let pd = matches.value_of("pd").unwrap();
+        let db = matches.value_of("db").unwrap();
+        let db_type = if db == "kv" { DBType::KV } else { DBType::RAFT };
```

Comment:
It would be good to throw an error if they user inputs something other than `kv` or `raft`.

Reply:
Yes, but this is fine since mostly we use tikv-ctl internally to debug, not those users.